packages/venia-concept/webpack.config.js

```diff
@@ -79,7 +79,10 @@ module.exports = async env => {
                 ? JSON.stringify(process.env.STORE_VIEW_CODE)
                 : JSON.stringify(storeConfigData.code),
             AVAILABLE_STORE_VIEWS: JSON.stringify(availableStores),
-            DEFAULT_LOCALE: JSON.stringify(global.LOCALE)
+            DEFAULT_LOCALE: JSON.stringify(global.LOCALE),
+            DEFAULT_COUNTRY_CODE: JSON.stringify(
+                process.env.DEFAULT_COUNTRY_CODE || 'US'
+            )
```

Comment:
In the event we do not have a `process.env.DEFAULT_COUNTRY_CODE`, this line will attempt to `JSON.stringify('US')`, which results in `"\"US\""`.

Instead, we want to do this like `STORE_VIEW_CODE` above:

```
DEFAULT_COUNTRY_CODE: process.env.DEFAULT_COUNTRY_CODE
  ? JSON.stringify(process.env.DEFAULT_COUNTRY_CODE)
  : 'US'
```

Reply:
Regarding `STORE_VIEW_CODE` in both cases in the ternary operator it's applying `JSON.stringify`.  
```
...
STORE_VIEW_CODE: process.env.STORE_VIEW_CODE
    ? JSON.stringify(process.env.STORE_VIEW_CODE)
    : JSON.stringify(storeConfigData.code),
...
```
Shall I ignore this implementation and apply the code changes that you've requested ?

Reply:
Oops, nevermind, this is a quirk of the webpack DefinePlugin. These values should work 👍 .